internal/pkg/aws/apprunner/service.go

```diff
@@ -33,6 +34,11 @@ type EnvironmentVariable struct {
 	Value string
 }
 
+type RuntimeEnvironmentSecret struct {
```

Comment:
Can we add doc string for public struct?
```suggestion
type EnvironmentSecret struct {
```

Reply:
Added to new commit.